### ISSUE TITLE
Button to cycle through comments by a given user

### ISSUE THREAD

**Issue by msirek** [Reviewable/Reviewable#909]
Often I would like to go back to an older comment someone made, but it is difficult to find in the older comments, or sometimes the comment disappears entirely (I've seen this when the comment is on a commit message which gets modified).  It would be useful to have a `reviewer` button, similar to the `unreviewed` or `unresolved` buttons, but with a drop-down arrow to select a specific user who has left comments.  Subsequent clicks of that button would cycle through all of the comments left by that user.

**Comment by pkaminski** [comment]
I agree with the general idea, but I think this should wait until we have a list of discussions in a sidebar.  Then we can add all kinds of filtering / searching feature there.

**Comment by brendanwalters** [comment]
In a large review, it is really hard to find my own (blocking) comments, being able to "next" through a particular reviewer's comments (or just my own) would be really helpful.

**Comment by pkaminski** [comment]
I think we can do even better with a general-purpose discussion-filtering UI.  :)
